frontend/src/components/common/QuestionCard.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import {
+  Box,
+  Container,
+  Flex,
+  Image,
+  Spacer,
+  HStack,
+  Tag,
+  Text,
+} from "@chakra-ui/react";
+
+type QuestionCardProps = {
+  date: Date;
+  questionTitle: string;
+  image: string;
+  text: string;
+  tags: string[];
+};
+
+const QuestionCard = ({
+  date,
+  questionTitle,
+  image,
+  text,
+  tags,
+}: QuestionCardProps): React.ReactElement => {
+  return (
+    <Container
+      borderRadius="22px"
+      padding="24px"
+      maxWidth="xl"
+      background="#fff"
+    >
+      <Flex>
+        <Text fontSize="16px" lineHeight="21px" align="left">
+          {`${date.getFullYear()} / ${String(date.getMonth() + 1).padStart(
+            2,
+            "0",
+          )} / ${String(date.getDate()).padStart(2, "0")}`}
+        </Text>
+        <Spacer />
+        <HStack display={["none", "block"]}>
+          <Text
+            as="button"
+            fontWeight="700"
+            fontSize="16px"
+            align="right"
+            marginRight="24px"
+          >
+            Edit
+          </Text>
+          <Text
+            as="button"
+            color="#CC4949"
+            fontWeight="700"
+            fontSize="16px"
+            align="right"
+          >
+            Delete
+          </Text>
+        </HStack>
+      </Flex>
+      <Box fontWeight="500" fontSize="24px" marginBottom="12px">
```

Comment:
just curious if these box tags are necessary? / does it work with just the Text tag?